package.json

```diff
@@ -4,7 +4,10 @@
   "description": "prettier plugin for solidity",
   "main": "src",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prettier": "prettier \"{.,{src,tests_config}/**}/*.{js{,on},md}\" \"tests/**/jsfmt.spec.js\"",
+    "lint": "npm run prettier -- --list-different",
```

Comment:
interesting, are we using prettier as a linter as well?

Reply:
Yep! It can’t hurt :)